aspnetcore/includes/razor.md

```diff
@@ -1,9 +1,10 @@
-This tutorial teaches ASP.NET Core MVC web development with controllers and views. Razor Pages is a feature of the ASP.NET Core MVC framework that makes building and testing web UI easier and more productive. You can use Razor pages alongside controllers and views in the same project.
+This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial.
 
-We recommend you try the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) tutorial before the MVC/Controller/Views version. The Razor Pages tutorial:
+Razor Pages:
 
-* Is the preferred approach for new application development.
-* Is easier to follow.
-* Covers more features.
+* Are part of the ASP.NET Core MVC framework. You can mix Razor Pages with controllers and views.
```

Comment:
```suggestion
* Are part of ASP.NET Core MVC. You can mix Razor Pages with controllers and views.
```

Reply:
MVC is more commonly referred to as an app model within ASP.NET Core.

Reply:
Waiting for @danroth27 	to decide language.